objects/src/accounts/package/storage_entry/template.rs

```diff
@@ -0,0 +1,115 @@
+use alloc::{
+    string::{String, ToString},
+    vec::Vec,
+};
+
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+use vm_core::{Felt, Word};
+use vm_processor::Digest;
+
+use crate::accounts::package::AccountComponentTemplateError;
+
+// TEMPLATE KEY
+// ================================================================================================
+
+// A simple wrapper type around a string key, used to enable templating.
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
+pub struct TemplateKey {
+    key: String,
+}
```

Comment:
The `TryFrom<&str>` impl expects braces but the `new` method doesn't. It would be good to clarify in the docs which method expects what kind of input. Note that the `fn try_from` can also be documented and it will show up in the rust docs.